tools/theme-preview.sh

```diff
@@ -196,6 +198,9 @@ _lp_jobcount() {
 _lp_python_env() {
   lp_python_env=pyenv
 }
+_lp_terraform_env() {
+  lp_terraform_env=stage
+}
```

Comment:
The point of the theme-preview file is to showcase the different types of information that Liquidprompt can gather, and how a specific theme displays it. Most themes, including how this patch makes the default theme display, will display a Terraform workspace the same as any other developer environment. This won't add any value to previewing a theme. I would remove the changes to this file.

Reply:
I've removed it, but this seems to be analogous to the python virtual environment example defined above it.